core/include/scipp/core/element/arithmetic.h

```diff
@@ -86,7 +94,12 @@ constexpr auto plus =
 constexpr auto minus =
     overloaded{add_types_t{}, [](const auto a, const auto b) { return a - b; }};
 constexpr auto times = overloaded{
-    times_types_t{}, [](const auto a, const auto b) { return a * b; }};
+    times_types_t{},
+    transform_flags::expect_no_in_variance_if_out_cannot_have_variance,
+    [](const auto a, const auto b) {
+      return a * b;
+    } // namespace scipp::core::element
+};
```

Comment:
Are the in-place versions supported already, or will this be added in the future?

Reply:
In place operations are already supported.